Bottom: 639709a8c0fdaa7815be10507f8efa3a53cd4bb0
Top:    170bd5a990ef76c111a06c974a5d93d8325ac0fd
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-27 17:22:41 +0000

Refresh of hid-gtk-remove-unused-vx2-and-

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 6b2dd87..9fd781c 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -33,21 +33,6 @@ static void zoom_by (double factor, int x, int y);
 
 int ghid_flip_x = 0, ghid_flip_y = 0;
 
-/* ------------------------------------------------------------ */
-
-static inline int 
-Vx2 (int x)
-{     
-  return (x - gport->view_x0) / gport->zoom + 0.5;
-}       
-        
-static inline int 
-Vy2 (int y)
-{     
-  return (y - gport->view_y0) / gport->zoom + 0.5;
-}       
-
-/* ------------------------------------------------------------ */
 
 static void
 ghid_pan_fixup ()
